kafka-connect/scripts/kafka_connect_run.sh

```diff
@@ -59,6 +59,9 @@ EOF
   cat /tmp/barnabas-connect.properties
   echo ""
 
+  # dir for saving application logs
+  export LOG_DIR=/tmp/logs
```

Comment:
Does anything actually read `LOG_DIR`? AFAICS, it's only `KAFKA_LOG_DIRS` which is actually used.

Reply:
This affects the logs (as in Log4j logs). Before I moved this to the /tmp directory Kafka Connect was complaining that it cannot create the logs directory in /opt/kafka (where the kafka user has no write access). This should stay there to avoid the error and should be removed with #21 once we stop logging to files.

Reply:
OK, thanks. I noticed only after I commented that you'd mentioned this problem in the PR description.